spk/sabnzbd/src/wizard/install_uifile

```diff
@@ -16,4 +16,9 @@
             }
         }]
     }]
+}, {
```

Comment:
You can remove this addition. The install wizard will only show up if it's a clean install of the package (i.e. no upgrade). In other words, it doesn't do anything for us ;)